### PR TITLE
Issue #6068: mention pocicli features in xdoc

### PR DESCRIPTION
**Pull request by romani**
Issue #6068

### CODE REVIEW

File: src/xdocs/cmdline.xml.vm [hunk 1]
```diff
@@ -167,6 +167,13 @@ java -D&lt;property&gt;=&lt;value&gt;  \
         specifying a property file using the <code>-p</code> option.  If a
         property file is specified, the system properties are ignored.
       </p>
+      <p>
+        CLI is implemented by means of <a href="https://picocli.info">picocli</a>
+        library Our CLI supports definition of arguments in file
```
Comment (rnveach) [inline src/xdocs/cmdline.xml.vm]: period was forgotten after `library`. See https://github.com/checkstyle/checkstyle/pull/6180#discussion_r229297295

Comment (romani) [reply src/xdocs/cmdline.xml.vm]: fixed. sorry.

**Comment by rnveach** [comment]
I am squashing and merging to master.

**Comment by rnveach** [comment]
Merged as 1cfe4ecfb8148c3364bb55a422e37ab36de6a89d

**Pull request closed without merging**